lib/ably/rest/client.rb

```diff
@@ -146,6 +150,7 @@ def initialize(options)
         @custom_host      = options.delete(:rest_host)
         @custom_port      = options.delete(:port)
         @custom_tls_port  = options.delete(:tls_port)
+        @add_request_ids  = options.delete(:add_request_ids) == false ? false : true
```

Comment:
As an optional extra debug option, this should default to false (unless and until it's added to the spec). So just `@add_request_ids  = options.delete(:add_request_ids)` is fine.

(Also, `cond ? false : true` is a pointless pattern, it's equivalent to `!cond`)